### PR TITLE
Feat: create product classes

### PR DESCRIPTION
**Pull request by FrancescoFantinel**

### CODE REVIEW

File: src/MiniERP.Models/ProductInOut.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```
Comment (Burilec) [inline src/MiniERP.Models/ProductInOut.cs:5]: Eh necessário esses usings

File: src/MiniERP.Models/ProductSummary.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```
Comment (Burilec) [inline src/MiniERP.Models/ProductSummary.cs:5]: Esses usings tbm

File: src/MiniERP.Models/ProductInOut.cs (hunk 1 above)
Comment (FrancescoFantinel) [reply src/MiniERP.Models/ProductInOut.cs]: Resolvido.

File: src/MiniERP.Models/ProductSummary.cs (hunk 2 above)
Comment (FrancescoFantinel) [reply src/MiniERP.Models/ProductSummary.cs]: Resolvido.

**Review by Burilec — APPROVED** [review]

**Pull request merged**